ui/src/components/domain/ManageDomains.js

```diff
@@ -482,7 +482,7 @@ class ManageDomains extends React.Component {
                             <TableHeadStyled align={center}>
                                 GCP Project ID
                             </TableHeadStyled>
-                            <TableHeadStyled align={left}>
+                            <TableHeadStyled align={center}>
```

Comment:
ah nice, was wondering why it was align "left" in the previous pr. Should've caught this on the previous review sorry about that